examples/index.md

```diff
@@ -246,15 +219,7 @@ Content-Type: application/vnd.api+json
     {
       "status": "403",
       "source": { "pointer": "data/attributes/secret-powers" },
-      "detail": "Access to secret powers not authorized on Sundays."
-    },
-    {
-      "status": "404",
-      "links": { "about": {
-        "href": "http://example.com/weapons/vorpal-blade",
-        },
-      },
-      "title": "Weapon not found."
+      "detail": "Editing secret powers is not authorized on Sundays."
```

Comment:
I changed this detail because _accessing_ an attribute can't be a 403, because an attribute doesn't have it's own URI. Instead, you'll have to request a whole resource, and then the 403 will apply to that whole request. Also, the only way I could see these error object realistically be returned together was on a request like `PATCH /wizards/1`, so "editing" made sense for the message.